client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -972,3 +968,19 @@ function getNamespaceDialog() {
 function isCacheStateChanged(prevProps, props) {
   return prevProps.cachedState !== props.cachedState;
 }
+
+function isPropertiesPanelLayoutChanged(prevProps, props) {
+
+  // same Object
+  if (props.layout === prevProps.layout) {
+    return false;
+  }
+
+  // deleted in one of the two
+  if (!props.layout || !prevProps.layout) {
+    return true;
+  }
+
+  // check JSON equality
+  return JSON.stringify(prevProps.layout.propertiesPanel) !== JSON.stringify(props.layout.propertiesPanel);
```

Comment:
In my manual test, `handlePropertiesPanelLayoutChange` and `componentDidUpdate` are called all the time, and `width` in `prevProps` and `props` is always different.

Reply:
This is caused by https://github.com/bpmn-io/bpmn-js-properties-panel/pull/923